.github/workflows/release.yml

```diff
@@ -17,12 +17,12 @@ jobs:
         run: sudo apt-get install zip
 
       - name: Create archives
-        run: for d in ./*; do if [ -d "$d" ]; then cd "$d"; zip ../"$d".cna *; cd ..; fi; done
+        run: sudo ./create_cnapy_projects_zip.sh
```

Comment:
Can you use sudo in a github action? And why?

Reply:
As far as I understand it should be possible since "sudo apt-get install zip" runs perfectly beforehand, if not I'll try to call the script directly and if this leads to permission problems (that's why I added the sudo, but the script doesn't need it) I'll convert it to a one-liner ^^